setup-py-project.md

```diff
@@ -0,0 +1,62 @@
+### 1. Add GitHub workflow
+The workflow [file](https://github.com/Flow-Launcher/plugin-samples/blob/master/HelloWorldPython/.github/workflows/Publish%20Release.yml) will help build and deploy your project, it does the following things:
+1. `workflow_dispatch:` gives you the option to manually run your workflow from the Actions section of your project
+2. On pushes to main, it will kick off the workflow but ignore the push if it's only changes made to the workflow file.
+```yml
+push:
+    branches: [ main ]
+    paths-ignore: 
+      - .github/workflows/*
+```
+3. It specifies the python version that will be used for building your project:
+```yml
+ strategy:
+      matrix:
+        python-version: [3.8]
+```
```

Comment:
Strategy is only useful for running multiple versions of Python. We should set an environmental variable of a single version.
```
env:
  python_ver: 3.8
```